### PR TITLE
Admin Agent Provisioning - functional tests

### PR DESCRIPTION
**Pull request by rayking29**
# Implements [AB#14637](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/14637)

## Admin Agent Provisioning
- created functional tests
- renamed queryString to searchString in AgentAccessService
- renamed data test ids

## Testing

- [ ] Unit Tests Updated
- [x] Functional Tests Updated
- [ ] Not Required

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

**Review by ytqsl — DISMISSED** [review]

File: Apps/Admin/Tests/Functional/cypress/integration/e2e/agentaccess.cy.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+describe("Provision", () => {
+    beforeEach(() => {
+        cy.login(
+            Cypress.env("keycloak_username"),
+            Cypress.env("keycloak_password"),
+            "/provision"
+        );
+    });
+
+    it.only("Create, search, update, delete user.", () => {
```
Comment (sslaws) [inline Apps/Admin/Tests/Functional/cypress/integration/e2e/agentaccess.cy.js:13]: This doesn't adhere to the typical set of tests that are validating discrete functionality.

**Review by sslaws — CHANGES_REQUESTED** [review]

File: Apps/Admin/Server/Program.cs [hunk 2]
```diff
@@ -104,6 +104,7 @@ public static async Task Main(string[] args)
 
         private static void AddModules(IServiceCollection services, IConfiguration configuration, ILogger logger, IWebHostEnvironment environment)
         {
+            ExceptionHandling.ConfigureProblemDetails(services, environment);
```
Comment (MikeLyttle) [inline Apps/Admin/Server/Program.cs:107]: This is a result of a merge conflict and shouldn't be included.

**Review by MikeLyttle — CHANGES_REQUESTED** [review]

**Review by ytqsl — APPROVED** [review]

File: Apps/Admin/Server/Program.cs (hunk 2 above)
Comment (BrianMaki) [reply Apps/Admin/Server/Program.cs:107]: Yes that shouldn't be there.  It was moved.

File: Apps/Admin/Tests/Functional/cypress/integration/e2e/agentaccess.cy.js (hunk 1 above)
Comment (rayking29) [reply Apps/Admin/Tests/Functional/cypress/integration/e2e/agentaccess.cy.js]: Same as communications.cy.js

File: Apps/Admin/Server/Program.cs (hunk 2 above)
Comment (rayking29) [reply Apps/Admin/Server/Program.cs]: Removed.

**Review by MikeLyttle — APPROVED** [review]

**Review by BrianMaki — APPROVED** [review]

**Review by ytqsl — APPROVED** [review]

**Pull request merged**